src/components/VerticalSectionOverflow/pageObject/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * VerticalSectionOverflow page object class.
+ * @class
+ */
+class PageVerticalSectionOverflow {
+    /**
+     * Create a new VerticalSectionOverflow page object.
+     * @constructor
+     * @param {string} rootElement - The selector of the VerticalSectionOverflow root element.
+     */
+    constructor(rootElement) {
+        this.rootElement = rootElement;
+    }
+
+    /**
+     * Clicks the vertical section overflow button.
+     * @method
+     */
+    click() {
+        $(this.rootElement).$('.rainbow-vertical-section-overflow_button').click();
+    }
+
+    /**
+     * Returns true when the overflow section is visible, false otherwise.
+     * @method
+     * @returns {bool}
+     */
+    isVerticalOverflowVisible() {
```

Comment:
only isVisible

Reply:
isVisible or isOpen?? or isExpended or is Collapsed